Source/MonoGame.Extended/Maps/Tiled/TiledTileLayer.cs

```diff
@@ -92,6 +92,29 @@ public override void Draw(RectangleF visibleRectangle)
             _spriteBatch.End();
         }
 
```

Comment:
Copying and pasting the whole method is not a very maintainable solution. Also, by removing the visible rectangle you've removed an important optimization for drawing large maps.